CONTRIBUTING.md

```diff
@@ -123,8 +203,8 @@ Before a pull request will be considered by the Azure SDK team, the following re
   - Short description of the payload of pull request.
 - After the pull request is submitted:
   - Our SLA is 48 hours. When your PR is submitted someone on our team will be auto assigned the PR for review. No need to email us
-  - MS internal folks, please reach out to us via our slack channel or
-  - Send an email to the Azure DevEx .Net team (azdevxdotnet@microsoft.com) alias.
+  - MS internal folks, please reach out to us via our Teams channel or
+  - Send an email to the Azure Developer Platform team [adpteam@microsoft.com](mailto:adpteam@microsoft.com) alias.
```

Comment:
@shahabhijeet @AlexGhiondea what would be a good alias here?

Reply:
We might also want to consider pointing folks at the Codeowners file instead of putting an alias here.

Reply:
What do you mean by CodeOwners file?

Reply:
We have plans to setup CODEOWNERS files in all our repos (see https://help.github.com/en/articles/about-code-owners), once that happens we should just refer to that file to find the correct owner to contact.